UnityProject/Assets/Scripts/Objects/Consoles/SyndicateOpConsole.cs

```diff
@@ -0,0 +1,112 @@
+using UnityEngine;
+using Managers;
+using Antagonists;
+using Strings;
+using System;
+using System.Collections.Generic;
+
+public class SyndicateOpConsole : MonoBehaviour
+{
+
+
+	public static SyndicateOpConsole Instance;
+
+	public int TcReserve;
+
+	public int TcIncrement = 14;
+
+	private bool warDeclared = false;
+	private bool rewardGiven = false;
+	private int timer = 1200;
+
+	private int timerIncrement = 60;
+	private int tcToGive = 280;
+
+	public List<SpawnedAntag> Operatives = new List<SpawnedAntag>();
+
+	public int Timer => timer;
+
+	private void Awake()
+	{
+		if ( Instance == null )
+		{
+			Instance = this;
+		}
+		else
+		{
+			Destroy(gameObject);
+		}
+	}
+
+
+	private void OnEnable()
+	{
+		if (CustomNetworkManager.IsServer)
+		{
+			UpdateManager.Add(ServerUpdateTimer, 1f);
+		}
+	}
+
+	private void OnDisable()
+	{
+		if (CustomNetworkManager.IsServer)
+		{
+			UpdateManager.Remove(CallbackType.PERIODIC_UPDATE, ServerUpdateTimer);
+		}
+	}
+
+
+	public void AnnounceWar(string DeclerationMessage)
+	{
+		if (warDeclared == false)
+		{
+			warDeclared = true;
+
+			GameManager.Instance.CentComm.ChangeAlertLevel(CentComm.AlertLevel.Red, true);
+			CentComm.MakeAnnouncement(ChatTemplates.PriorityAnnouncement, 
+			$"Attention all crew! An open message from the syndicate has been picked up on local radiowaves! Message Reads:\n" +
+			$"{DeclerationMessage}" ,CentComm.UpdateSound.Alert);
+
+			var antagplayers = AntagManager.Instance.CurrentAntags;
+
+			foreach (var antag in antagplayers )
+			{
+				if (antag.Antagonist.AntagJobType == JobType.SYNDICATE)
+				{
+					Operatives.Add(antag);
+				}
+			}
+		}
+	}
+
+	public void ServerUpdateTimer()
+	{
+		if (warDeclared == false || rewardGiven) return;
+
+		if (timerIncrement > 0 || timer > 0)
```

Comment:
this timer is weird, why the need for two variables? Can't we just count seconds and update the UI or whatever this is used with division and modulo?

Reply:
What I really meant is, this would make its intention clearer.

Reply:
the timer variable is used directly for the gui
the timer increment var is used to count to 60

Reply:
what happens when you finish counting to 60?

Reply:
I assume tc gets paid out

Reply:
it pays out TC equivelent to Tc Increment
unless there is not enough tc to give then itll just give the rest of the tc in reserve

Reply:
Oh yeah, I can see that in the else. In plain English, how should the timer work?

Reply:
one timer counts down from 1200 and the other counts down from 60
probably should use modulo or smth instead